src/main/java/core/basesyntax/FindNumbers.java

```diff
@@ -11,6 +14,13 @@
      * числа 92, 18, 26 и 0.</p>
      */
     public int[] getAllNumbers(String text) {
-        return null;
+        String[] numbersString = text.split("([^-0-9]+)");
+        List<Integer> list = new ArrayList<>();
```

Comment:
let's use `.replaceAll()` as well when processing `text`

and let`s not use List, only arrays